gen/dcos-config.yaml

```diff
@@ -407,9 +407,20 @@ package:
   - path: /etc/dcos-log.env
     content: |
       DCOS_LOG_CONFIG_PATH=/opt/mesosphere/etc/dcos-log-config.json
-  - path: /etc/dcos-log-config.json
+  - path: /etc_master/dcos-log-config.json
     content: |
       {
+        "role": "master"
+      }
+  - path: /etc_slave/dcos-log-config.json
+    content: |
+      {
+        "role": "agent"
+      }
+  - path: /etc_slave_public/dcos-log-config.json
+    content: |
+      {
+        "role": "agent_public"
```

Comment:
As far as I understand this, it looks good to me. Where are these new role definitions used exactly?